gatekeeper.ino

```diff
@@ -25,17 +25,19 @@ void loop() {
   if (stanBramy == LOW) {
     lcd.setCursor(7, 0);
     lcd.print("OPEN    ");
-  } else
-  lcd.setCursor(7, 0);
```

Comment:
On Java only the first line after else is executed.
Same for arduino (See [Example Code comment for IF](https://www.arduino.cc/reference/en/language/structure/control-structure/if/)).
This is why I put brackets here.